strapi/strapi
# fix(content-releases): Send the populate entry when publish

### What does it do?

Entity manager needs the complete and populated entry when publishing, so we need to send entries populated to handle validation on components.

### How to test it?

1. Create a release and add an entry with components and these components should have some validation errors
2. Try to publish the release
3. It should return an error
